api/src/main/resources/api/service_openapi.yaml

```diff
@@ -1204,6 +1204,25 @@ components:
               items:
                 type: string
                 enum: ["IS_MEMBER", "PATH", "NUM_CHILDREN", "IS_ROOT"]
+        includeRelationshipFields:
```

Comment:
Why are these fields not just part of the hierarchy, especially when the hierarchy is a parameter anyways?

Reply:
I did it this way because we need a way to get per-node rollups that don't involve a hierarchy. Open to other suggestions though.

Reply:
Also, the `relatedEntity` property doesn't apply to the hierarchy fields (is_root, etc.)

Reply:
I see. I guess this is fine.